VTUL/dlp-access
# LIBTD-2382: Add sort options to search page and sort configurations for searchPage in admin

**admin- sort config for search**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2382) (:star:)

# What does this Pull Request do? (:star:)
This PR adds sort options to search results configuration menu in search page as well as adding sort configurations for searchPage config in admin site.

# What's the changes? (:star:)

* admin_searchpage_facet_sort_config.spec.js: Added tests for sort configurations
* browse_collections.spec.js, search_bar_spec.js: Adjusted tests in browse collections page and search bar in search page
* SearchLoader.js, SearchResults.js: Added sort options to search page
* SiteAdmin.js, SearchForm.js: Added sort configs for searchPage in admin site
* ResultsNumberDropdown.js, CollectionsListPage.scss, ListPages.scss, SearchResult.scss: css style changes

# How should this be tested?

* Go to "Search" page and see if the search results can be sorted by title (ascending or descending order) or date
* Go to "searchpage config" in "SiteAdmin" page and see if the sort options can be modified (add/update/delete) and be reflected on the "Search" page

# Interested parties
Tag (@yinlinchen) interested parties

(:star:) Required fields
